libs/explorers/ui/src/lib/components/footer/footer.component.scss

```diff
@@ -0,0 +1,73 @@
+@use 'libs/explorers/styles/src/lib/mixins';
+
+footer {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--color-primary);
+  padding: 10px;
+
+  @include mixins.respond-to('medium') {
+    justify-content: space-between;
+  }
+
+  .footer-links {
+    ul {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      list-style: none;
+      gap: 40px;
```

Comment:
Colocated certain size related properties like gap and font-size.  Colors can be variables but size-related concerns are colocated.